scrape-util-projects/occupant-query/config.toml

```diff
@@ -0,0 +1,31 @@
+[acquire.static.settings]
+is-active = true
+source = '/var/www/www-data/occquse/responses/'
+on-raw = '/var/local/scrape-util/projects/occupant-query/raw'
```

Comment:
I believe this should named after `survey_display` since you renamed it a while back.